scripts/remote_customize_entrypoint.sh

```diff
@@ -113,6 +137,26 @@ done
 set -x
 echo on
 
+# if dkms was used, unmount the extra dirs before packaging
+if [ "$is_dkms" = "true" ]; then
+  echo "Unmounting system directories"
+  if umount /mnt/image/image-root/sys; then
+    echo "Unmounted /sys"
+  else
+    echo "Failed to Unmount /sys"
```

Comment:
Nit: to be consistent with the other messages

```suggestion
    echo "Failed to unmount /sys"
```